http-core/src/main/mima-filters/1.2.x.backwards.excludes/content-media-type-value-string.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+ProblemFilters.exclude[ReversedMissingMethodProblem]("org.apache.pekko.http.javadsl.model.ContentType.value")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("org.apache.pekko.http.javadsl.model.MediaType.value")
```

Comment:
This is only justified if there are no user types extending from these classes (or their subclasses), i.e. if these traits are marked `@DoNotInherit`. The traits are not marked `@DoNotInherit`, so naively you'd say this change is not allowed. However, I think you can make the argument that the fact that `@DoNotInherit` is missing from this trait is an oversight, and it was never intended to be user-extended. It would be good to add the `@DoNotInherit` now, then?

Reply:
Do you mean putting `@DoNotInherit` on `ContentType`/`MediaType` traits themselves?

Reply:
No thats fair, I thought you were talking about `ValueRenderable` but that wouldn't make sense its already marked as `@InternalApi`.

We can remove and also fix the Java namings (i.e. `value` should be `getValue`) as well as removing `@DoNotInherit` in 2.x.x